src/sharedServices/isDateOverDaysLimit.spec.ts

```diff
@@ -0,0 +1,48 @@
+import { isDateOverDaysLimit } from './isDateOverDaysLimit'
+import { currentDateEpochMilliseconds } from '../utils/currentDateEpochMilliseconds'
+import { when } from 'jest-when'
+jest.mock('../utils/currentDateEpochMilliseconds', () => ({
+  currentDateEpochMilliseconds: jest.fn()
+}))
+
+describe('isDateOverDaysLimit', () => {
+  const MOCK_CURRENT_TIME = 1666956229830
+  const DAYS_LIMIT = 7
+  it('should return true if date is 1 millisecond over the limit', () => {
+    when(currentDateEpochMilliseconds).mockReturnValue(MOCK_CURRENT_TIME)
```

Comment:
Is it worth doing this in a beforeEach hook so we don't have to do write it at the start of each test?

Reply:
That's been moved.